stl/inc/atomic

```diff
@@ -2626,46 +2626,44 @@ _Ty atomic_fetch_xor_explicit(
 
 #if _HAS_CXX20
 _EXPORT_STD template <class _Ty>
-void atomic_wait(const volatile atomic<_Ty>* const _Mem, const typename atomic<_Ty>::value_type _Expected) noexcept
-/* strengthened */ {

```

Comment:
I always thought these comments were intended to be with all of the noexcepts.

Reply:
The "strengthened" comment means that "the Standard doesn't depict/guarantee `noexcept` here, but we've added `noexcept` to our implementation as permitted by the Standard". If the Standard is later updated to add `noexcept`, then we're simply conforming to it, not going beyond the Standard, so we remove "strengthened" comments at that time.